.taskcluster.yml

```diff
@@ -288,10 +288,73 @@ tasks:
                 description: ${job.name}
                 owner: ${owner}
                 source: ${repo.url}
+          releaseTasks:
+          - taskId: {$eval: 'as_slugid("release")'}
+            provisionerId: proj-taskcluster
+            workerType: release
+            created: {$fromNow: ''}
+            deadline: {$fromNow: '3 hours'}
+            extra:
+              notify:
+                email:
+                  subject: 'Taskcluster Release Status'
+            routes:
+            - notify.email.taskcluster-notifications@mozilla.com.on-any
+            - notify.irc-channel.#taskcluster-bots.on-any
+            scopes:
+            - secrets:get:project/taskcluster/release
+            payload:
+              features:
+                taskclusterProxy: true
+              maxRunTime: 3600
+              command:
+                - - "/bin/bash"
+                  - "-ec"
+                  - |-
+                    # set up golang..
+                    export GOROOT="$(pwd)/go/go"
+                    export GOPATH="$(pwd)/go/path"
+                    mkdir -p $${GOPATH}
+                    export PATH="$${GOROOT}/bin:$${GOPATH}/bin:$${PATH}"
+                    go version
+
+                    # set up gox
+                    go get github.com/mitchellh/gox
+
+                    # set up Node, using nvm (generic-worker does not support .xz, so we can't use mounts)
+                    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
+                    . ~/.nvm/nvm.sh
+                    nvm install ${node}
+                    npm install --global yarn
```

Comment:
I can't find it right now but I feel like I've seen lots of bug reports in yarn over the years based on being installed with npm rather than their recommended way. I don't think we need to be _particularly_ worried about the cautionary note in [their docs](https://yarnpkg.com/en/docs/install#alternatives-stable), but we may as well do it their recommended way.

Reply:
trying it now..